config/upgrade_config.py

```diff
@@ -0,0 +1,110 @@
+import attr
```

Comment:
How long do you think we'll need to keep this script around?

Reply:
Hopefully only for 1-2 releases. Or perhaps it should be released with Trainer 1.0 and removed after that.